config/Dockerfiles/cloud-image-compose/cloud-image-compose.sh

```diff
@@ -32,6 +32,13 @@ virtlogd &
 
 chmod 666 /dev/kvm
 
+# If rpm_repo is local, need to start local http server to host rpms
+if [[ $rpm_repo == /* ]]; then
+    pushd dirname $rpm_repo
```

Comment:
did you mean
`pushd $(dirname $rpm_repo)`

Reply:
ty